default/stringtables/en.txt

```diff
@@ -11617,8 +11617,8 @@ LRN_PSY_DOM
 Psychogenic Domination
 
 LRN_PSY_DOM_DESC
-'''Has a 10% chance of taking control of enemy ships with non-telepathic crews in the same system as a planet with domination focus, unless the enemy empire also knows the [[tech LRN_PSY_DOM]].
-Protects against [[predefinedshipdesign SM_PSIONIC_SNOWFLAKE]]s taking over your ships.
+'''Has a 10% chance of taking control of enemy ships with non-telepathic crews in the same system as a planet with domination focus, unless the enemy empire also knows the [[tech LRN_PSY_DOM]]. Domination focus can only be set on planets colonized by telepathic species.
+Reduces the chance that [[predefinedshipdesign SM_PSIONIC_SNOWFLAKE]]s take over your ships.
```

Comment:
Preferably also avoid referring to the player in the second person ("you" / "your")